unstructured/ingest/pipeline/reformat/chunking.py

```diff
@@ -62,3 +85,49 @@ def run(self, elements_json: str) -> Optional[str]:
 
     def get_path(self) -> Path:
         return (Path(self.pipeline_context.work_dir) / "chunked").resolve()
+
+    def chunk(self, elements_json_file: str) -> Optional[list[Element]]:
+        """Called by Chunker.run() to properly execute the defined chunking_strategy."""
+        # -- No chunking_strategy means no chunking --
+        if self.chunking_config.chunking_strategy is None:
+            return
+        # -- Chunk locally --
+        if self.chunking_config.chunking_strategy in ("basic", "by_title"):
+            return dispatch.chunk(
+                elements=elements_from_json(filename=elements_json_file),
+                chunking_strategy=self.chunking_config.chunking_strategy,
+                combine_text_under_n_chars=self.chunking_config.combine_text_under_n_chars,
+                include_orig_elements=self.chunking_config.include_orig_elements,
+                max_characters=self.chunking_config.max_characters,
+                multipage_sections=self.chunking_config.multipage_sections,
+                new_after_n_chars=self.chunking_config.new_after_n_chars,
+                overlap=self.chunking_config.overlap,
+                overlap_all=self.chunking_config.overlap_all,
+            )
+        # -- Chunk remotely --
+        if self.partition_config.partition_by_api:
+            return partition_via_api(
+                filename=elements_json_file,
+                # -- If api_key or api_url are None, partition_via_api will raise an error, which
+                # -- will be caught and logged by Chunker.run()
+                api_key=self.partition_config.api_key,  # type: ignore
+                api_url=self.partition_config.partition_endpoint,  # type: ignore
+                chunking_strategy=self.chunking_config.chunking_strategy,
+                combine_under_n_chars=self.chunking_config.combine_text_under_n_chars,
+                # -- These are not currently supported by the sdk and raise an error.
+                # -- They will need to be updated once we bump the client version.
+                # combine_text_under_n_chars=self.chunking_config.combine_text_under_n_chars,
+                # include_orig_elements=self.chunking_config.include_orig_elements,
+                max_characters=self.chunking_config.max_characters,
+                multipage_sections=self.chunking_config.multipage_sections,
+                new_after_n_chars=self.chunking_config.new_after_n_chars,
+                overlap=self.chunking_config.overlap,
+                overlap_all=self.chunking_config.overlap_all,
+            )
+        # -- Warn that the defined chunking_strategy is not locally available --
+        logger.warning(
+            "There is no locally available chunking_strategy:"
+            f" {self.chunking_config.chunking_strategy}."
+            "If trying to partition remotely, check that `partition_by_api`, `api_url`,"
+            "and `api_key` are correctly defined."
```

Comment:
When using a multi-line string like this, make them all `f-` strings if any of them need to be.

Also, remember that continuation lines should start with the space that separates the segments. In this case there are no spaces between line 2 and 3 or 3 and 4.

Placing the spaces all at the front of lines 2+ makes it easy to see they're all there because they're all lined up in a column. Including the `f` prefix helps with that too.